CHANGES.md

```diff
@@ -320,6 +322,8 @@ To be released.
 [#1343]: https://github.com/planetarium/libplanet/pull/1343
 [#1348]: https://github.com/planetarium/libplanet/pull/1348
 [#1351]: https://github.com/planetarium/libplanet/pull/1351
+[#1363]: https://github.com/planetarium/libplanet/issues/1363
```

Comment:
Could you explain why this PR related to #1363 
Isn't it related to #1361 and #1362 instead?

Reply:
Thanks for your review. Amended in https://github.com/planetarium/libplanet/pull/1365/commits/0b325f9351fc8d051176bb72c34b91939b7f3b1c